requirements/base.txt

```diff
@@ -30,5 +30,6 @@ python-slugify==4.0.1     # via cookiecutter
 requests==2.24.0          # via cookiecutter
 six==1.15.0               # via astroid, cookiecutter, edx-lint, python-dateutil
 text-unidecode==1.3       # via python-slugify
+typed-ast==1.4.1          # via astroid
```

Comment:
Hmm, looks like the upgrade job is still on 3.5, which is why there's this odd, solitary change -- I'll update the upgrader settings.